README.md

```diff
@@ -59,7 +59,7 @@ npm install @google-cloud/compute
 
 ```javascript
 /**
- * TODO(developer): Uncomment these variables before running the sample.
+ * TODO(developer): Uncomment and replace these variables before running the sample.
```

Comment:
I think changes in the README will get overwritten though because it's from a template.